extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/filter/OperationFilter.java

```diff
@@ -97,6 +100,20 @@ private String methodRef(Operation operation) {
         return (String) (extensions != null ? extensions.get(EXT_METHOD_REF) : null);
     }
 
+    private void maybeAdd400Response(Map.Entry<PathItem.HttpMethod, Operation> operation, String methodRef) {
```

Comment:
A name like addBadRequestResponseIfInputMethod would spell out exactly what’s happening, making things clearer for anyone reading the code later.